tests/system/src/test/java/test/robot/javafx/scene/SliderTooltipNPETest.java

```diff
@@ -0,0 +1,151 @@
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package test.robot.javafx.scene;
+
+import java.util.concurrent.CountDownLatch;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.control.Slider;
+import javafx.scene.control.Tooltip;
+import javafx.scene.input.MouseButton;
+import javafx.scene.robot.Robot;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
+import javafx.stage.WindowEvent;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import test.util.Util;
+
+/*
+ * Unit test for verifying Slider NPE error.
+ *
+ * There is 1 tests in this file.
+ * Steps for testSliderTooltipNPE()
+ * 1. Create a slider and tooltip.
+ * 2. Make setAutoHide of tooltip as true and add tooltip to slider.
+ * 3. Hover over slider thumb and wait for tooltip. Drag the thumb.
+ * 4. Verify that NullPointerException is not thrown.
+ */
+public class SliderTooltipNPETest {
+    static CountDownLatch startupLatch = new CountDownLatch(1);
+    static volatile Throwable exception;
+    static volatile Stage stage;
+    static volatile Scene scene;
+    static Robot robot;
+    static Slider slider;
+    static final int SCENE_WIDTH = 250;
+    static final int SCENE_HEIGHT = SCENE_WIDTH;
+    final int DRAG_DISTANCE = 10;
+
+    public static void main(String[] args) {
+        initFX();
+        try {
+            SliderTooltipNPETest test = new SliderTooltipNPETest();
+            test.testSliderTooltipNPE();
+        } catch (Throwable e) {
+            e.printStackTrace();
+        } finally {
+            exit();
+        }
+    }
+
+    @Test
+    public void testSliderTooltipNPE() throws Throwable {
+        dragSliderAfterTooltipDisplayed(DRAG_DISTANCE, true);
+    }
+
+    private void dragSliderAfterTooltipDisplayed(int dragDistance, boolean xIncr) throws Throwable {
```

Comment:
`boolean xIncr` is not required for the test. It can be removed along with it's use on line 101.